types/dispatchr/addons/createStore.d.ts

```diff
@@ -0,0 +1,13 @@
+import { StoreClass, Store } from '../index';
+
+interface StoreOptions {
+    initialize?(): void;
+    storeName: string;
+    handlers: { [event: string]: string };
+    [prop: string]: any;
```

Comment:
This index property shouldn't be needed; the type of `CreateStore` _should_ allow extra properties.

Missing `dehydrate`, `hydrate`, `statics` and `mixins`.